pkg/apiserver/archive/archive.go

```diff
@@ -57,7 +58,7 @@ func Register(r *gin.RouterGroup, s *Service) {
 	endpoint.GET("", s.list)
 	endpoint.GET("/detail", s.detail)
 	endpoint.GET("/report", s.report)
-	endpoint.DELETE("/:uid", s.delete)
+	endpoint.DELETE("", s.delete)
```

Comment:
Can we support both `DELETE /:uid` and `Delete /?uids=x,x,x` because I think It's better to keep the restful style.

Reply:
I will test later in my local env.

Reply:
After testing, I found that it can be done.

Reply:
Let me see, there are some minor changes.